test/junit/scala/tools/nsc/QuickfixTest.scala

```diff
@@ -6,25 +6,30 @@ import org.junit.Test
 import java.nio.file.Files
 import scala.reflect.internal.util.BatchSourceFile
 import scala.reflect.io.AbstractFile
+import scala.tools.nsc.reporters.StoreReporter
 import scala.tools.testkit.BytecodeTesting
 import scala.tools.testkit.ReleasablePath._
 import scala.util.Using
 
-import reporters.StoreReporter
```

Comment:
I thought of you when I typed that at midnight, knowing you'd want me to use more keys on the keyboard.

Reply:
I like IntelliJ's Organize Imports myself. My opinions about imports don't go any deeper than that.